### ISSUE TITLE
How are data split during fine-tuning?

### ISSUE THREAD

**Issue by adosar** [hspark1212/MOFTransformer#165]
First of all, congratulations for this excellent work!

I have a question regarding the data split during the fine-tuning step. In the [paper](https://chemrxiv.org/engage/api-gateway/chemrxiv/assets/orp/resource/item/63c74dbb07308a37cb9c864f/original/a-multi-modal-pre-training-transformer-for-universal-transfer-learning-in-metal-organic-frameworks.pdf) it is mentioned that:

> The fine-tuning dataset was randomly split into training, validation, test sets in a ratio of 0.8:0.1:0.1.

From Figure 3, it can be seen that the dataset size is varied between 5k, 10k, 15k and 20k. For each of these sizes `s`, is the following correct: `0.8*s` are used for training, `0.1*s` for validation and `0.1*s` for test? Meaning that the `x-axis != train_size`?

**Comment by Yeonghun1675** [comment]
Hi @adosar,
You're right. The data size is the number of data including train, val, and test. That is, the number of each dataset multiplied by 0.8 was used as training data.
